lib/geordi/commands/tests.rb

```diff
@@ -1,14 +1,40 @@
-desc 'tests', 'Run all employed tests'
-def tests
-  rake_result = invoke_geordi 'with_rake'
-
-  # Since `rake` usually is configured to run all tests, only run them if `rake`
-  # did not perform
-  if rake_result == :did_not_perform
-    invoke_geordi 'unit'
-    invoke_geordi 'rspec'
-    invoke_geordi 'cucumber'
-  end
+desc 'tests [FILES and OPTIONS]', 'Run all employed tests'
+long_desc <<-LONGDESC
+When running `geordi tests` with a test file or a test directory, it will
+invoke either `geordi rspec` or `geordi cucumber` to run the test, depending on
+whether the first argument is a spec or a feature.
+
+Command line arguments are passed to `geordi rspec`/`geordi cucumber` as well.
+
+When running `geordi tests` without any arguments, all unit tests, rspec specs
+and cucumber features will be run.
+LONGDESC
+
+def tests(*args)
+  if args.any?
+    args, opts = Thor::Options.split(args)
 
-  Interaction.success 'Successfully ran tests.'
+    if args.empty?
+      Interaction.fail "geordi tests only supports arguments if one or more test files/directories are specified."
+    elsif args.first.start_with? 'spec'
+      invoke 'rspec', args, opts
+    elsif args.first.start_with? 'features'
+      invoke 'cucumber', args, opts
+    else
+      Interaction.fail "The first argument #{args.first} could not be classified as either a rspec or a cucumber test."
```

Comment:
I think this error message should be more precise. It hints that we try to parse file types which we don't. And I think we should also tell users how to use it correctly. Something like  
```
geordi tests cannot process the argument #{args.first}. The first argument has to be a path beginning with "features" or "spec". 
```